rusk-wallet/src/wallet.rs

```diff
@@ -1128,14 +1146,21 @@ impl<F: SecureWalletFile + Debug> Wallet<F> {
         Err(Error::Unauthorized)
     }
 
+    /// Check if the address is stored in our profiles, return the address if
+    /// found
+    pub fn claim_as_address(&self, addr: Address) -> Result<Address, Error> {
```

Comment:
what do you think about calling this `is_owned` and have it return a bool? Then we avoid movine the address around.

Reply:
I thought about this, and returning a bool would shade the error in the find_index

Additionally, returning the address improves the usability (imo) for our usecases 
```rust
                let addr = match address {
                    Some(addr) => wallet.claim_as_address(addr)?,
                    None => wallet.default_address(),
                };
``` 

Since I'm here, I'll probably change `claim_as_address` into `claim` (because it's already an `Address`)